.eslintignore

```diff
@@ -1,10 +1,9 @@
 lib/internal/v8_prof_polyfill.js
 lib/punycode.js
-test/addons/??_*/
+test/addons/??_*
```

Comment:
So how exactly does this make the ignoring more correct?

Reply:
It's an [`ignore`](https://github.com/kaelzhang/node-ignore) thing, it builds the exclusion regex based on this line (requiring a terminal `/`), then the directory itself is matched as `test/addons/12_XFD` (with no trailing `/`) so it passes and all it's children are enumerated (only they  fail).
So the line pre PR is equivalent to `test/addons/??_*/*` and `test/addons/??_*` make it exclude the directories and not even enumerate their content.